packages/mdc-notched-outline/README.md

```diff
@@ -2,24 +2,27 @@
 title: "Notched Outline"
 layout: detail
 section: components
-excerpt: "The notched outline is a border around either a text field or select element"
+excerpt: "The notched outline is a border around either a Text Field or Select element"
 iconId: text_field
 path: /catalog/input-controls/notched-outline/
 -->
 
 # Notched Outline
 
-The outline is a border around all sides of either a text field or select component. This is used for the Outlined variant of either a Text Field or Select.
+The notched outline is a border around all sides of either a Text Field or Select component. This is used for the Outlined variant of either a Text Field or Select.
```

Comment:
@kfranqueiro should I remove this `outlined variant` select since we currently don't support it?